client/azure_client_set.go

```diff
@@ -3,9 +3,9 @@ package client
 import (
 	"fmt"
 
-	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-06-01/compute"
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-07-01/compute"
 	"github.com/Azure/azure-sdk-for-go/services/dns/mgmt/2017-10-01/dns"
```

Comment:
What about other APIs, for example DNS here? Any specific reason not to update them?

There is a newer Service API version https://github.com/Azure/azure-sdk-for-go/tree/master/services/dns/mgmt/2018-05-01/dns

Reply:
I was focusing on the ones that could yield bigger impact. Should I update all of them? Maybe that's a good idea.

Reply:
I would go for all of them :)